drivers/scheduler/k8s/k8s.go

```diff
@@ -618,6 +660,16 @@ func (k *k8s) DeleteVolumes(ctx *scheduler.Context) ([]*volume.Volume, error) {
 				}
 			}
 			logrus.Infof("[%v] Destroyed PVC: %v", ctx.App.Key, obj.Name)
+		} else if obj, ok := spec.(*snap_v1.VolumeSnapshot); ok {
+			if err := k8sOps.DeleteSnapshot(obj.Metadata.Name, obj.Metadata.Namespace); err != nil {
+				if matched, _ := regexp.MatchString(".+ not found", err.Error()); !matched {
```

Comment:
nit: can we just use strings.Contains() here as well as above?

Reply:
Done